### PR TITLE
Feature/hacker get

### PR DESCRIPTION
**Pull request by YiFeiZhang2**
Creat get route for hacker by id.

### CODE REVIEW

File: routes/api/hacker.js [hunk 1]
```diff
@@ -122,6 +122,31 @@ module.exports = {
...
+                    "data": {...}
+                }
+
+         * @apiError {String} message Error message
+         * @apiError {Object} data empty
+         * @apiErrorExample {object} Error-Response: 
+         *      {"message": "Issue with retrieving hacker information", "data": {}}
+         */
+        hackerRouter.route("/:id").get(
+            Controllers.Hacker.findById
```
Comment (pierreTklein) [inline routes/api/hacker.js]: lmao no validation?

File: tests/hacker.test.js [hunk 2]
```diff
@@ -15,6 +15,25 @@ const storedHacker1 = util.hacker.HackerA;
 const newHacker1 = util.hacker.newHacker1;
 const invalidHacker1 = util.hacker.invalidHacker1;
 
+describe("GET hacker", function () {
```
Comment (pierreTklein) [inline tests/hacker.test.js]: For the future, look into testing failure cases too

File: routes/api/hacker.js (hunk 1 above)
Comment (YiFeiZhang2) [reply routes/api/hacker.js]: Deferred

File: tests/hacker.test.js (hunk 2 above)
Comment (YiFeiZhang2) [reply tests/hacker.test.js]: Deferred

**Pull request merged**